UI/Source/CollectionViews/mac/CollectionViewHostItem.swift

```diff
@@ -21,7 +21,9 @@ public final class CollectionViewHostItem: NSCollectionViewItem {
         }
         didSet {
             // Only add the view if it isn't already added (i.e. hit the noop case in `willSet`.)
-            if let theView = hostedView as? NSView , theView.superview != view {
+            if let view = hostedView as? NSView, newValue == view {
```

Comment:
willSet looks good but this didSet doesn't compile since newView doesn't exist in in didSet and the definition of `theView` is now missing.

I think this didSet logic can actually stay as is. It could be changed to `if let view = hostedView as? NSView, view != oldValue {` but it seems reasonable to me to just ensure that the superview is correct as the code was doing before since thats what matters. The important part is the change you made to willSet.

Reply:
derp. Yea just compiled Pilot instead of PilotUI... hold on